### PR TITLE
Issue 5536: (SLTS) - Validate provided config values are within acceptable range.

### LINKED ISSUE #5536: SLTS - ChunkedSegmentStorageConfig should validate provided values are within acceptable range.
**Problem description**
ChunkedSegmentStorageConfig does not throw exceptions when invalid values like negative numbers etc are specified as config.

**Problem location**
ChunkedSegmentStorageConfig 

**Suggestions for an improvement**
should validate provided values are within acceptable range.

### PR DESCRIPTION
**Pull request by sachin-j-joshi**
Signed-off-by: Sachin Joshi <sachin.joshi@emc.com>

**Change log description**  
Validate provided config values are within acceptable range.

**Purpose of the change**  
Fixes #5536

**What the code does**  
Validate provided config values are within acceptable range.

**How to verify it**  
Unit tests should pass.

### CODE REVIEW

File: segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorageConfig.java [hunk 1]
```diff
@@ -203,4 +203,36 @@
     public static ConfigBuilder<ChunkedSegmentStorageConfig> builder() {
         return new ConfigBuilder<>(COMPONENT_CODE, ChunkedSegmentStorageConfig::new);
     }
+
+    private int getNonNegativeInt(TypedProperties properties, Property<Integer> property) {
```
Comment (andreipaduroiu) [inline segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorageConfig.java:207]: Let's put these on `TypedProperties` and have it throw `InvalidPropertyValueException` just like its counterparts.

**Review by andreipaduroiu — CHANGES_REQUESTED** [review]

File: segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorageConfig.java (hunk 1 above)
Comment (RaulGracia) [reply segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorageConfig.java:207]: Agree, this functionality may be useful for other classes, so we can put these methods on a place at which they could be easily reused.

**Review by RaulGracia — COMMENTED** [review]
Looks good, just place the new methods on another location at which they could be reused.

**Comment by andreipaduroiu** [comment]
@sachin-j-joshi Are we still pursuing this?

**Pull request closed without merging**